example/android/app/src/main/kotlin/com/example/example/MainActivity.kt

```diff
@@ -1,6 +0,0 @@
-package com.example.example
-
-import io.flutter.embedding.android.FlutterActivity
-
-class MainActivity: FlutterActivity() {
```

Comment:
why did you delete an example activity?

Reply:
Because it's code ruins codefactor and without it app works correctly